packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -51,84 +51,297 @@ const WrappedCalendar = onClickOutside(Calendar)
+     */
     maxDate: PropTypes.object,
+    /**
+     * Minimum value of date that can be selected by user.
+     */
     minDate: PropTypes.object,
+    /**
+     * Prop to show multiple months on date picker.
+     */
     monthsShown: PropTypes.number,
+    /**
+     * Value for name of custom input.
+     */
     name: PropTypes.string,
+    /**
+     * A callback function to execute when object loses focus.
+     */
     onBlur: PropTypes.func,
+    /**
+     * A callback function to execute when date is selected.
+     */
     onChange: PropTypes.func.isRequired,
+    /**
+     * A callback function to execute when value is selected from dropdown.
+     */
     onSelect: PropTypes.func,
+    /**
+     * A callback function to execute when week is selected.
+     */
     onWeekSelect: PropTypes.func,
+    /**
+     * A callback function to execute when the mouse cursor leaves the element.
+     */
     onClickOutside: PropTypes.func,
+    /**
+     * A callback function to execute when date is entered.
+     */
     onChangeRaw: PropTypes.func,
+    /**
+     * A callback function to execute when object focused.
+     */
     onFocus: PropTypes.func,
+    /**
+     * A callback function to execute when a key is pressed.
+     */
     onKeyDown: PropTypes.func,
+    /**
+     * A callback function to execute when month is selected.
+     */
     onMonthChange: PropTypes.func,
+    /**
+     * Prop to open calendar on a particular date.
+     */
     openToDate: PropTypes.object,
+    /**
+     * A prop to check next month dates.
+     */
     peekNextMonth: PropTypes.bool,
+    /**
+     * Value for placeholder of date picker.
+     */
     placeholderText: PropTypes.string,
+    /**
+     * Prop to prevent date picker from opening on focus.
+     */
     preventOpenOnFocus: PropTypes.bool,
+    /**
+     * Prop to make date picker as read only.
+     */
     readOnly: PropTypes.bool,
+    /**
+     *Prop to make the date picker as a required field.
+     */
     required: PropTypes.bool,
+    /**
+     * Prop to allow scrollable year dropdown.
+     */
     scrollableYearDropdown: PropTypes.bool,
+    /**
+     * Selected Date Value.
+     */
     selected: PropTypes.object,
+    /**
+     * Prop to select end date on a date picker.
+     */
     selectsEnd: PropTypes.bool,
+    /**
+     * Prop to select start date on a date picker.
+     */
     selectsStart: PropTypes.bool,
+    /**
+     * Prop to show month drop down in calendar.
+     */
     showMonthDropdown: PropTypes.bool,
+    /**
+     * Prop to show week numbers .
+     */
     showWeekNumbers: PropTypes.bool,
+    /**
+     * Prop to show a dropdown to select year in date picker calendar .
+     */
     showYearDropdown: PropTypes.bool,
+    /**
+     * Prop to show a month navigation in date picker .
+     */
     forceShowMonthNavigation: PropTypes.bool,
+    /**
+     * Date selected by user .
+     */
     startDate: PropTypes.object,
```

Comment:
This is the start date using in a date range picker. The description for this prop should be similar to `endDate`.

Reply:
Resolved in 9519de4